src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -11300,18 +11292,10 @@ namespace HVACUnitarySystem {
 						CompOffMassFlow = UnitarySystem( UnitarySysNum ).CoolMassFlowRate( CoolSpeedNum - 1 );
 						CompOffFlowRatio = UnitarySystem( UnitarySysNum ).MSCoolingSpeedRatio( CoolSpeedNum - 1 );
 					}
-				} else {
-					if ( CoolSpeedNum < 1 ) {
-						CompOffMassFlow = UnitarySystem( UnitarySysNum ).IdleMassFlowRate;
-						CompOffFlowRatio = UnitarySystem( UnitarySysNum ).IdleSpeedRatio;
-					} else if ( CoolSpeedNum == 1 ) {
-						if ( UnitarySystem( UnitarySysNum ).MultiSpeedCoolingCoil ) {
-							CompOffMassFlow = 0.0; // #5518
-							CompOffFlowRatio = 0.0;
-						} else {
-							CompOffMassFlow = UnitarySystem( UnitarySysNum ).IdleMassFlowRate;
-							CompOffFlowRatio = UnitarySystem( UnitarySysNum ).IdleSpeedRatio;
-						}
+				} else { // cycling fan mode
+					if ( CoolSpeedNum <= 1 ) {
+						CompOffMassFlow = 0.0; // #5518
+						CompOffFlowRatio = 0.0;
```

Comment:
I guess you decided that speednum didn't need to be split out like this anymore? (https://github.com/NREL/EnergyPlus/pull/5519/files#r54980671)

Reply:
Yes, the VS and MS coils should behave the same at speed = 1 for cycling fan mode. #5518 addressed a multi-speed coil issue and I didn't think much about the VS coil at that time until I saw this same problem with VS coils. I then concluded that VS and MS should be treated the same when speed = 1 and the fan cycles (i.e., should cycle between x m3/s and 0 if cycling fan, not x m3/s and idle air flow rate).

Reply:
:+1: